### PR TITLE
private link guide

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by fiquick** [comment]
[request add a detailed guide of how to enable privatelink on Aura Console](https://trello.com/c/aSWv9vZj/340-request-add-a-detailed-guide-of-how-to-enable-privatelink-on-aura-console)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 1]
```diff
@@ -205,13 +205,48 @@ It will be different from the *Connection URI* you used before.
 
 image::azure_privatelink_03_browser_bloom_over_vpn.png["Accessing Browser and Bloom over a VPN", title="Accessing Browser and Bloom over a VPN"]
 
-==== Enabling private endpoints
+==== Enabling Azure private endpoints step-by-step
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:208]:
```suggestion
==== Enable Azure private endpoints
```

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 2]
```diff
@@ -205,13 +205,48 @@ It will be different from the *Connection URI* you used before.
...
 
 To enable private endpoints using Azure Private Link:
 
 . Select *Network Access* from the sidebar menu of the Console.
 . Select *New network access configuration* and follow the setup instructions.
 
+*Configure Network Access*
+
+. Choose your product from the available options.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:217]:
```suggestion
. Select your product from the available options.
```

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 3]
```diff
@@ -205,13 +205,48 @@ It will be different from the *Connection URI* you used before.
...
+. Select the appropriate region for your deployment.
+. Enter the "Target Azure Subscription ID" where you want to enable PrivateLink.
+. Enable PrivateLink by toggling the respective option.
+
+*Obtain PrivateLink Service Name*
+
+. After enabling PrivateLink, you will receive a PrivateLink service name.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:224]:
```suggestion
. After enabling PrivateLink, you receive a PrivateLink service name.
```

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 4]
```diff
@@ -205,13 +205,48 @@ It will be different from the *Connection URI* you used before.
...
+
+*Create PrivateLink Endpoint in the Azure portal*
+
+. Log in to your Azure portal.
+. Navigate to your cloud VPC and create a new PrivateLink endpoint.
+. Use the PrivateLink service name obtained in the previous step for the configuration.
+
+*Accept Endpoint in Aura Console*
+
+. Return to the Aura Virtual Dedicated Cloud Console.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:235]:
```suggestion
. Return to the Aura Console.
```
The console is the console, right? VDC is a tier. Unsure of the terminology here.

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 5]
```diff
@@ -205,13 +205,48 @@ It will be different from the *Connection URI* you used before.
...
+. Check for the newly created PrivateLink endpoint.
+. Accept the endpoint to complete the connection process.
+
+*Disable Public Traffic*
+
+. Before disabling public traffic, test all your application connectivity with PrivateLink to ensure everything is functioning correctly.
+. Once verified, you can disable public traffic by toggling off the public access option.
+. Note: You can always disable public traffic later if needed.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:243]:
```suggestion
. Note: If needed, you can postpone disabling public traffic.
```

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 6]
```diff
@@ -205,13 +205,48 @@ It will be different from the *Connection URI* you used before.
...
+
+*Monitor PrivateLink Status*
+
+. You can monitor the status of your PrivateLink configuration on the Aura Virtual Dedicated Cloud Console.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:247]:
```suggestion
. You can monitor the status of your PrivateLink configuration in the Aura Console.
```

**Review by AlexicaWright — COMMENTED** [review]
Some comments. It's slightly odd to only have instructions for Azure and not AWS or GCP.
On an editorial note, I think you should change _Enabling private endpoints_ to _Enable ..._ in the other two sub-headers also.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 7]
```diff
@@ -205,12 +205,41 @@ It will be different from the *Connection URI* you used before.
 
 image::azure_privatelink_03_browser_bloom_over_vpn.png["Accessing Browser and Bloom over a VPN", title="Accessing Browser and Bloom over a VPN"]
 
-==== Enabling private endpoints
-
-To enable private endpoints using Azure Private Link:
-
-. Select *Network Access* from the sidebar menu of the Console.
-. Select *New network access configuration* and follow the setup instructions.
+==== Enabling Azure Private Endpoints for Aura
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:208]:
```suggestion
==== Enable Azure Private Endpoints for Aura
```
For consistency ;)

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 8]
```diff
@@ -205,12 +205,41 @@ It will be different from the *Connection URI* you used before.
...
+
+. To enable private endpoints using Azure Private Link:
+.. Go to the sidebar menu in the Aura console, then select *Security > Network Access > Network Access*
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:211]:
```suggestion
.. From the sidebar menu in the Aura console, select *Security > Network Access > Network Access*
```

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 9]
```diff
@@ -205,12 +205,41 @@ It will be different from the *Connection URI* you used before.
...
+.. Select *New network access configuration* and follow the setup instructions.
+
+. Configure Network Access in the Aura console
+.. Select your product from the available options.
+.. Select the appropriate region for your deployment. (Azure Private Link will apply to all instances in the region.)
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:216]:
```suggestion
.. Select the appropriate region for your deployment. (Azure Private Link applies to all instances in the region.)
```

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 10]
```diff
@@ -205,12 +205,41 @@ It will be different from the *Connection URI* you used before.
...
+.. Select *Enable Private Link*. 
+
+. Obtain a Private Link service name
+.. After enabling Private Link, you receive a Private Link service name in the Aura console.
+.. Copy this service name, you need it in the next step.
+
+. Create Private Link endpoint in the Azure portal
+.. Log in to your Azure portal.
+.. Navigate to your cloud VPC and create a new Private Link endpoint.
+.. Use the Private Link service name obtained in the step 3 for the configuration.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:227]:
```suggestion
.. Use the Private Link service name obtained in step three for the configuration.
```

**Review by AlexicaWright — COMMENTED** [review]
Looks great! Just a few small editorial suggestions. Well done!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-515.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-515.surge.sh/aura/platform/security/secure-connections)

**Pull request merged**